src/yb/master/master.proto

```diff
@@ -1996,6 +1996,14 @@ message FlushTablesResponsePB {
 
   optional bytes flush_request_id = 2;
 }
+message FlushSysCatalogRequestPB{
+
+}
+message FlushSysCatalogResponsePB{
+  optional MasterErrorPB error = 1;
+
+  optional bytes flush_request_id = 2;
```

Comment:
do we use this field anywhere?

Reply:
Actually not yet. I look at the codes for FlushTablesResponsePB, which has these two fields and at first write out the same.

Reply:
After fix the code in catalog_manager.cc to return the status, these fields might be used